### ISSUE TITLE
list results seem to be unmapped unexpectedly

### ISSUE THREAD

**Issue by rntz** [talonvoice/beta#129]
Consider https://github.com/knausj85/knausj_talon/blob/master/code/keys.py#L238

If I replace all the uses of captures/angle brackets, `<self.letter>`, with uses of the corresponding lists/curly brackets, `{self.letter}`, then `unmodified_key` stops working, because it produces things that haven't been mapped through the list. For example, spoken "one" produces "one" rather than "1".

The result of annotating the code with a print statement, `print(m, str(m), m[0])` is:

```
2020-10-13 13:00:16    IO DECODED ['control', 'one']
2020-10-13 13:00:16    IO one one one
```

I'd like to minify this but I haven't had time yet.

**Comment by lunixbochs** [comment]
Should be fixed in next beta.

**Comment by rntz** [comment]
Seems fixed as of talon 0.1.2-76.
